### PR TITLE
authorization for edit link

### PR DESCRIPTION
**Pull request by keith-ferney**
this should hide the link and the edit link seems to already redirect to login if the user is not logged in and canvas is setup

### CODE REVIEW

File: app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -197,14 +197,19 @@ def course_list
   end
 
   def update
+    if session[:lms_authenticated_user] | @document.organization[:enable_anonymous_actions] | has_role("designer")
```
Comment (llamapope) [inline app/controllers/documents_controller.rb]: are these binary or? should it be `||`

Comment (keith-ferney) [reply app/controllers/documents_controller.rb]: yea it should. good thing you caught that but when i tested, it was working i will fix this tomorrow

File: app/controllers/documents_controller.rb [hunk 2]
```diff
@@ -13,19 +13,23 @@ def index
   end
 
   def new
-    @document = Document.new(name: 'Unnamed')
+    if session[:lms_authenticated_user] || @organization[:enable_anonymous_actions] || has_role("designer")
```
Comment (llamapope) [inline app/controllers/documents_controller.rb]: move these to a function - can_use_edit_token or something like that

**Comment by llamapope** [comment]
create pull request to stage

**Pull request closed without merging**